skillmap/src/components/InfoPanel.tsx

```diff
@@ -83,12 +83,13 @@ export class InfoPanelImpl extends React.Component<InfoPanelProps> {
         const isMap = !node;
         const isActivity = node && !isRewardNode(node);
         const tags = isActivity && (node as MapActivity).tags || undefined;
+        const thumbnailAltText: string = `Thumbnail for ${!subtitle ? "starting" : title} activity in skillmap ${!subtitle ? title : subtitle}`;
```

Comment:
we need to pass these through our localization pipeline so the text makes sense for non-english speaking users. This is done through the 'lf' function, and would look like 

```suggestion
        const thumbnailAltText = lf("Thumbnail for {0} activity in skillmap {1}", !subtitle ? lf("starting") : title, !subtitle ? title : subtitle);
```